couch/datadog_checks/couch/couch.py

```diff
@@ -79,6 +80,18 @@ def get_server(self, instance):
             raise Exception("A server must be specified")
         return server
 
+    def get_config_tags(self, instance):
+        tags = instance.get('tags', [])
+
+        # Clean up tags in case there was a None entry in the instance
+        # e.g. if the yaml contains tags: but no actual tags
+        if tags is None:
```

Comment:
the if/else/return is a little too verbose, not a fan of oneliners but this might actually be
```
return list(set(tags)) if tags else []
```